fatiando/gravmag/transform.py

```diff
@@ -141,6 +141,8 @@ def upcontinue(x, y, data, shape, height):
 
     .. note:: x, y, z and height should be in meters.
 
+    .. note:: It is not possible to get the FFT of a masked grid. The default `gridder.interp()` call using minimum curvature will not be suitable. Use `extrapolate=True` or `algorithm='nearest'`.
```

Comment:
@mtb-za could you please break this into multiple lines? It's better to read (specially here on Github) if lines don't exceed 80 characters. You can do this in rst format:

    .. note::
        A long multiline
        note about some things.